woocommerce/woocommerce
# Updated Product attributes placeholder to e.g. length or weight

Closes #51134 .



### How to test the changes in this Pull Request:




1. Install **WooCommerce** and go to **Products > Add New**
2. Scroll down to the "Product data" meta box and click the **Attributes tab**
3. Click the Add new button and notice the **"f.e. size or color"** inside the name input field.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Changed Product attributes placeholder to e.g. length or weight
</details>
